src/commands/react-native/upload.ts

```diff
@@ -249,6 +255,27 @@ export class UploadCommand extends Command {
     })
   }
 
+  private resolveConfig = async () => {
+    try {
+      this.config = await parseConfigFile(this.config, this.configPath || DEFAULT_CONFIG_PATH)
```

Comment:
nit: No need for the `DEFAULT_CONFIG_PATH` fallback, it is already handled by `parseConfigFile()`

Reply:
I'm removing it in the next PR: https://github.com/DataDog/datadog-ci/pull/580 as part of refactoring, so let's change it then :)